app/models/optimized_image.rb

```diff
@@ -93,7 +93,7 @@ def self.create_for(upload, width, height, opts = {})
         opts = opts.merge(quality: target_quality) if target_quality
         opts = opts.merge(upload_id: upload.id)
 
-        if upload.extension == "svg"
+        if extension == ".svg" && upload.extension == "svg"
```

Comment:
I think just `extension == ".svg"` is enough here? We can't convert any raster images into an SVG anyways.

Reply:
I think this is a good point I guess we should reject earlier...